docs/DeveloperGuide.md

```diff
@@ -36,3 +44,64 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+## Parser Component
```

Comment:
Should these sections (Parser Component, Timetable Component and Next Command) be under the Design & Implementation sections instead of under the Instructions for manual testing section?